app/Resources/views/base.html.twig

```diff
@@ -32,7 +32,7 @@
         </div>
         <div id="navbar" class="collapse navbar-collapse">
           <ul class="nav navbar-nav">
-            <li class="active"><a href="/">Home</a></li>
+            <li class="active"><a href=" {{ path('home')}} ">Home</a></li>
```

Comment:
zróbmy tutaj z użyciem translatora, szkoda zeby to było na żywca w twigu trzymane